src/main/java/net/fabricmc/tinyremapper/TinyRemapper.java

```diff
@@ -1181,7 +1215,12 @@ public void run() {
 		/**
 		 * Propagate names into methods and create additional bridges to keep the normally mapped method name intact.
 		 */
-		COMPATIBLE;
+		COMPATIBLE
+	}
+
+	@Override
+	public ClassHeader getByName(String internalName) {
+		return this.defaultState.classes.get(internalName);
```

Comment:
This isn't really MRJ compatible, apply is MRJ aware, so the environment may be better served by MrjState?.